math/stan/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            stan-dev stan 2.31.0 v
+categories              math
+maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
+license                 BSD
+description             C++ package for Bayesian inference, maximum likelihood estimation etc.
+long_description        ${description}
```

Comment:
`${description}` needs to be expanded here: `{*}${description}`

Reply:
Oops, sorry again, I had this port for quite a while, missed to update this part. Will do now.